pkg/realmd/operation.html

```diff
@@ -8,6 +8,15 @@ <h4 class="modal-title realms-op-title"></h4>
         <div class="realms-op-fields">
           <p class="realms-op-leave-only-row"></p>
           <table class="form-table-ct realms-op-form">
+            <tr class="realms-op-error" hidden>
+              <td colspan="3">
+                <div class="alert alert-danger">
+                  <span class="pficon pficon-error-circle-o"></span>
+                  <strong class="realms-op-message"></strong>
+                </div>
+                <pre class="realms-op-diagnostics" hidden></pre>
+              </td>
+            </tr>
```

Comment:
Thanks! Having it at the bottom is consistent with the rest of Cockpit, even if it isn't with PatternFly.

If the change wasn't arbitrary, I would guess the reason why it's at the bottom in Cockpit is because it shifts all the elements down a bit — possibly even when you're interacting with them, and that would be a jarring experience when trying to interact with the dialog.

(My guess for why it's at the top in PatternFly is that the error might apply to the whole dialog. Both error message placements have their pros and cons.)